app/views/layout.nunjucks

```diff
@@ -29,7 +29,7 @@
 
     <link rel="stylesheet" href="{{ SITE_ROOT }}/css/print.css{{ qs }}" media="print" type="text/css">
 
-    <script type="text/javascript" src="{{ SITE_ROOT }}/js/vendor/polyfills/details.polyfill.js"></script>
+    <script type="text/javascript" src="{{ SITE_ROOT }}/js/vendor/polyfills/details.polyfill.js{{ qs }}"></script>
```

Comment:
The version of polyfill is best being contained in the file name. For the time being it would be fine to just rename the file to `details.polyfill.min.js`. This indicates the file is minified and follows the convention put in place with the `jQuery` file.